README.md

```diff
@@ -487,6 +487,17 @@ The TypedData class is a wrapper class for the serialized binary data. It presen
 
 TypedData does late deserialization of the data only when the get method is called.
 
+This TypedData allows you to retrieve the data type of the underlying binary to be used when being deserialized. This class represents the type of a Hazelcast serializable object. The fields can take the following values:
```

Comment:
There should be line break after this paragraph, otherwise the numbered list will not be formatted.